api-spec.md

```diff
@@ -6,7 +6,139 @@ The STAC API defines a RESTful JSON-based server to query [SpatioTemporal Asset
 (STAC). While the core STAC specification provides a structure and language to describe assets, users usually want to access
 a subset of the entire catalog, such as for a certain date range, in a particular area of interest, or matching properties
 they care about. STAC API specifies those query parameters, and compliant servers return collections of STAC Items that
-match the user's preferences.
+match the user's preferences. Most of STAC API's mechanics are specified by [OGC API](https://ogcapi.ogc.org/) family of
+standards, particularly [OGC API - Features](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html) (OAFeat). 
+
+## Overview
+
+The core of STAC API simply returns a valid [STAC Catalog](stac-spec/catalog-spec/catalog-spec.md) and a description
+of what parts of the fuller STAC API specification it conforms to. The `links` section of the Catalog is the jumping
+off point for the more powerful capabilities - it contains a list of URL's, each described by particular link 
+'relationships' (`rel`) to indicate their functionality. 
+
+The `search` rel is one of the most common, often located at a `/search` endpoint. It re-uses all of the OAFeat [query
+parameters](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_items_) specified in their 'core', and adds a 
+couple more. It does not require a full implementation of OAFeat, it is instead a simplified construct that can run
+a search across any set of indexed STAC [`Items`](stac-spec/item-spec/README.md). 
+
+The other common most common link relationship is `data`, which goes to a complete list of available 'Collections', along
+with a mechanism to request individual collections by ID. These are specified by OGC API, in the 
```

Comment:
I'm not clear on what this means "along with a mechanism to request individual collections by ID"
Does it mean that by using the `data` rel type you are implementing the /collections endpoint?

Reply:
Yes. I was attempting to make it so things specified the actual endpoints less. But I think OGC spells out the actual endpoint here, so I should probably just say it directly. 

I do think this is an area where we need clarity on 'browse' vs 'ogc'. I see in earth search you use child to go to collections. Which I think makes sense. I would have thought OpenEO would do the same, but they use data and collections/ endpoint. @m-mohr told me a reason that made sense, I think perhaps it's that with a strict STAC thing /collections/ should just be a bunch of child links - it shouldn't inline the actual collection information, like @matthewhanson does in https://earth-search.aws.element84.com/v0/collections